flow/flow.go

```diff
@@ -433,7 +439,7 @@ const reportMbits = false
 
 var sizeMultiplier uint
 var schedTime uint
-var hwtxchecksum bool
+var hwtxchecksum, hwrxpacketstimestamp bool
```

Comment:
If you don't use it outside Init function it is better to make it local like schedulerOffRemove := args.PersistentClones. Other variables are used global.